From a7e4c97b8b8e7e914e4a91e38810de498b6c0227 Mon Sep 17 00:00:00 2001
From: Ferdinand Thiessen <rpm@fthiessen.de>
Date: Wed, 12 May 2021 00:43:18 +0200
Subject: [PATCH] Allow setting custom DOCDIR

By setting DOC_DIR when invoking qmake you can now
set the target directory for the documentation files.
---
 qt-fsarchiver-terminal/qt-fsarchiver-terminal.pro | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/qt-fsarchiver-terminal/qt-fsarchiver-terminal.pro b/qt-fsarchiver-terminal/qt-fsarchiver-terminal.pro
index 4a2c1cb..bb2e8f5 100644
--- a/qt-fsarchiver-terminal/qt-fsarchiver-terminal.pro
+++ b/qt-fsarchiver-terminal/qt-fsarchiver-terminal.pro
@@ -102,7 +102,11 @@ SOURCES += src/archinfo.c \
            src/thread_comp.c \
            src/writebuf.c
 # install
+ isEmpty(DOC_DIR) {
+   DOC_DIR = /usr/share/doc/qt-fsarchiver-terminal
+ }
+
  target.path = /usr/sbin
- doc.files = doc
- doc.path = /usr/share/doc/qt-fsarchiver-terminal/doc
+ doc.files = doc/*
+ doc.path = $$DOC_DIR
 INSTALLS = target doc
-- 
2.26.2

